userspace/sysdig/sysdig.cpp

```diff
@@ -1147,17 +1096,28 @@ sysdig_init_res sysdig_init(int argc, char **argv)
 		plugins.add_directory(SYSDIG_PLUGINS_DIR);
 		plugins.read_plugins_from_dirs(inspector.get());
 
+		// Load container plugin
+		std::string container_config = R"({"hooks":["create","start"],"engines":{"docker":{"enabled":true,"sockets":["/var/run/docker.sock"]},"podman":{"enabled":true,"sockets":["/run/podman/podman.sock","/run/user/1000/podman/podman.sock"]},"containerd":{"enabled":false,"sockets":["/run/containerd/containerd.sock"]},"cri":{"enabled":true,"sockets":["/run/crio/crio.sock", "/run/containerd/containerd.sock"]},"lxc":{"enabled":false},"libvirt_lxc":{"enabled":false},"bpm":{"enabled":false}}})";
+		auto container_config_file = "/etc/sysdig/container.json";
+		if (std::filesystem::exists(container_config_file))
+		{
+			std::ifstream file(container_config_file);
+			std::stringstream buffer;
+			buffer << file.rdbuf();
+			container_config = buffer.str();
+		}
+
+		plugins.load_plugin(inspector.get(), "container");
+		plugins.config_plugin(inspector.get(), "container", container_config);
```

Comment:
IMO we could better factor out this logic, maybe in `plugin_utils.cpp` to avoid repetitions between `sysdig.cpp` and `csysdig.cpp`